peekingduck/configs/draw/legend.yml

```diff
@@ -0,0 +1,6 @@
+input: ["all"]
+output: ["img"]
+
+all_legend_items: ["fps", "count"]
+position: "bottom"
+include: ["all"]
```

Comment:
If I'm not wrong, this is not the same `["all"]` as input type, and can cause confusion for users. It won't work to make it ["fps", "count"] as well. Perhaps something like this would be clearer?
```suggestion
include: ["all_legend_items"]
```

Reply:
Done as suggested. Also changed node to use the all_legend_items